Add windows support

sodre opened the pull request.

sodre committed:
[win] Add windows to build scripts

  - Bumped build number
  - Moved sha256 into project body
  - grep is quiet

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

sodre commented:
@conda-forge-admin please rerender

conda-forge-admin committed:
MNT: Re-rendered with conda-smithy 3.2.2 and pinning 2018.12.18

sodre commented:
@conda-forge-admin, please rerender

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to re-render for you, but it looks like there was nothing to do.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

sodre committed:
[not win] Fix build.sh issues

  - Undefined OS variable.
  - Create activate-liquidprompt file during buildtime.
  - Only activate liquidprompt in interactive shells.

sodre commented:
@conda-forge-admin, please rerender
